docs/security.md

```diff
@@ -45,6 +45,7 @@ authentication:
     authorizationEndpoint:
     jwkEndpoint:
     redirectUrl:
+    redirectWebUrl: 
```

Comment:
where/how is this property used?

Reply:
If the web UI is started independently, you can specify the redirect path. 

By default, this configuration does not need to be specified.